2021-09-18 16:31:51,425 INFO    MainThread:3618044 [internal.py:wandb_internal():91] W&B internal server running at pid: 3618044, started at: 2021-09-18 16:31:51.424181
2021-09-18 16:31:51,426 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: check_version
2021-09-18 16:31:51,426 INFO    WriterThread:3618044 [datastore.py:open_for_write():80] open: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/run-g9jtz9zf.wandb
2021-09-18 16:31:51,428 DEBUG   SenderThread:3618044 [sender.py:send():179] send: header
2021-09-18 16:31:51,428 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: check_version
2021-09-18 16:31:51,461 DEBUG   SenderThread:3618044 [sender.py:send():179] send: run
2021-09-18 16:31:51,524 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:31:51,524 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:31:52,810 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:31:52,810 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:31:54,904 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:31:54,905 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:31:54,905 ERROR   SenderThread:3618044 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 133, in execute
    six.reraise(*sys.exc_info())
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 719, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 127, in execute
    return self.client.execute(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 39, in execute
    request.raise_for_status()
  File "/usr/local/lib/python3.6/dist-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql
2021-09-18 16:31:59,683 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:31:59,683 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:07,898 INFO    SenderThread:3618044 [dir_watcher.py:__init__():168] watching files in: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files
2021-09-18 16:32:07,898 INFO    SenderThread:3618044 [sender.py:_start_run_threads():719] run started: g9jtz9zf with start time 1631997110
2021-09-18 16:32:07,898 DEBUG   SenderThread:3618044 [sender.py:send():179] send: summary
2021-09-18 16:32:07,899 INFO    SenderThread:3618044 [sender.py:_save_file():841] saving file wandb-summary.json with policy end
2021-09-18 16:32:07,901 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: run_start
2021-09-18 16:32:08,901 INFO    Thread-8  :3618044 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/wandb-summary.json
2021-09-18 16:32:09,229 DEBUG   HandlerThread:3618044 [meta.py:__init__():39] meta init
2021-09-18 16:32:09,229 DEBUG   HandlerThread:3618044 [meta.py:__init__():53] meta init done
2021-09-18 16:32:09,229 DEBUG   HandlerThread:3618044 [meta.py:probe():210] probe
2021-09-18 16:32:09,235 DEBUG   HandlerThread:3618044 [meta.py:_save_pip():57] save pip
2021-09-18 16:32:09,237 DEBUG   HandlerThread:3618044 [meta.py:_save_pip():71] save pip done
2021-09-18 16:32:09,237 DEBUG   HandlerThread:3618044 [meta.py:probe():252] probe done
2021-09-18 16:32:09,240 DEBUG   SenderThread:3618044 [sender.py:send():179] send: files
2021-09-18 16:32:09,240 INFO    SenderThread:3618044 [sender.py:_save_file():841] saving file wandb-metadata.json with policy now
2021-09-18 16:32:09,246 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: stop_status
2021-09-18 16:32:09,247 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: stop_status
2021-09-18 16:32:09,303 ERROR   Thread-11 :3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:09,304 ERROR   Thread-11 :3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:09,304 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:09,305 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:09,901 INFO    Thread-8  :3618044 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/requirements.txt
2021-09-18 16:32:09,901 INFO    Thread-8  :3618044 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/output.log
2021-09-18 16:32:09,902 INFO    Thread-8  :3618044 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/wandb-metadata.json
2021-09-18 16:32:10,384 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:10,384 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:10,384 ERROR   SenderThread:3618044 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 133, in execute
    six.reraise(*sys.exc_info())
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 719, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 127, in execute
    return self.client.execute(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 39, in execute
    request.raise_for_status()
  File "/usr/local/lib/python3.6/dist-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql
2021-09-18 16:32:10,396 ERROR   Thread-11 :3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:10,397 ERROR   Thread-11 :3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:10,397 ERROR   Thread-11 :3618044 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 133, in execute
    six.reraise(*sys.exc_info())
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 719, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 127, in execute
    return self.client.execute(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 39, in execute
    request.raise_for_status()
  File "/usr/local/lib/python3.6/dist-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql
2021-09-18 16:32:12,681 ERROR   Thread-11 :3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:12,682 ERROR   Thread-11 :3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:12,817 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:12,817 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:17,346 INFO    Thread-11 :3618044 [upload_job.py:push():137] Uploaded file /tmp/3288847.1.gpu_ded.q/tmprpevqdg0wandb/176pugub-wandb-metadata.json
2021-09-18 16:32:31,910 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: stop_status
2021-09-18 16:32:31,911 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: stop_status
2021-09-18 16:32:37,641 DEBUG   SenderThread:3618044 [sender.py:send():179] send: stats
2021-09-18 16:32:42,478 DEBUG   SenderThread:3618044 [sender.py:send():179] send: telemetry
2021-09-18 16:32:42,479 DEBUG   SenderThread:3618044 [sender.py:send():179] send: exit
2021-09-18 16:32:42,479 INFO    SenderThread:3618044 [sender.py:send_exit():287] handling exit code: 1
2021-09-18 16:32:42,479 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:42,479 INFO    SenderThread:3618044 [sender.py:send_exit():295] send defer
2021-09-18 16:32:42,479 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:42,480 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:42,480 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 0
2021-09-18 16:32:42,480 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:42,480 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 0
2021-09-18 16:32:42,481 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 1
2021-09-18 16:32:42,481 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:42,481 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 1
2021-09-18 16:32:42,533 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:42,533 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 1
2021-09-18 16:32:42,533 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 2
2021-09-18 16:32:42,533 DEBUG   SenderThread:3618044 [sender.py:send():179] send: stats
2021-09-18 16:32:42,534 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:42,537 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 2
2021-09-18 16:32:42,537 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:42,537 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 2
2021-09-18 16:32:42,538 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 3
2021-09-18 16:32:42,538 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:42,538 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 3
2021-09-18 16:32:42,539 DEBUG   SenderThread:3618044 [sender.py:send():179] send: summary
2021-09-18 16:32:42,539 INFO    SenderThread:3618044 [sender.py:_save_file():841] saving file wandb-summary.json with policy end
2021-09-18 16:32:42,539 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:42,539 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 3
2021-09-18 16:32:42,539 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 4
2021-09-18 16:32:42,540 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:42,540 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 4
2021-09-18 16:32:42,540 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:42,540 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 4
2021-09-18 16:32:42,581 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:42,603 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:42,603 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:42,914 INFO    Thread-8  :3618044 [dir_watcher.py:_on_file_modified():229] file/dir modified: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/wandb-summary.json
2021-09-18 16:32:43,739 ERROR   SenderThread:3618044 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-18 16:32:43,739 ERROR   SenderThread:3618044 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-18 16:32:46,254 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 5
2021-09-18 16:32:46,254 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:46,255 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:46,255 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 5
2021-09-18 16:32:46,255 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:46,256 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 5
2021-09-18 16:32:46,256 INFO    SenderThread:3618044 [dir_watcher.py:finish():282] shutting down directory watcher
2021-09-18 16:32:46,360 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:46,916 INFO    SenderThread:3618044 [dir_watcher.py:_on_file_modified():229] file/dir modified: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/config.yaml
2021-09-18 16:32:46,916 INFO    SenderThread:3618044 [dir_watcher.py:finish():312] scan: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files
2021-09-18 16:32:46,917 INFO    SenderThread:3618044 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/wandb-metadata.json wandb-metadata.json
2021-09-18 16:32:46,917 INFO    SenderThread:3618044 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/config.yaml config.yaml
2021-09-18 16:32:46,917 INFO    SenderThread:3618044 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/requirements.txt requirements.txt
2021-09-18 16:32:46,917 INFO    SenderThread:3618044 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/wandb-summary.json wandb-summary.json
2021-09-18 16:32:46,921 INFO    SenderThread:3618044 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/output.log output.log
2021-09-18 16:32:46,924 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 6
2021-09-18 16:32:46,924 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:46,929 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:46,929 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 6
2021-09-18 16:32:46,930 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:46,930 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 6
2021-09-18 16:32:46,930 INFO    SenderThread:3618044 [file_pusher.py:finish():177] shutting down file pusher
2021-09-18 16:32:47,031 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:47,031 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:47,133 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:47,134 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:47,226 INFO    Thread-13 :3618044 [upload_job.py:push():137] Uploaded file /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/requirements.txt
2021-09-18 16:32:47,236 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:47,236 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:47,274 INFO    Thread-12 :3618044 [upload_job.py:push():137] Uploaded file /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/config.yaml
2021-09-18 16:32:47,326 INFO    Thread-14 :3618044 [upload_job.py:push():137] Uploaded file /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/files/wandb-summary.json
2021-09-18 16:32:47,338 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:47,338 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:47,440 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:47,441 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:47,526 INFO    Thread-7  :3618044 [sender.py:transition_state():308] send defer: 7
2021-09-18 16:32:47,527 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:47,527 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 7
2021-09-18 16:32:47,527 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:47,527 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 7
2021-09-18 16:32:47,543 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:47,580 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 8
2021-09-18 16:32:47,580 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:47,580 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:47,580 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 8
2021-09-18 16:32:47,581 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:47,581 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 8
2021-09-18 16:32:47,581 INFO    SenderThread:3618044 [sender.py:transition_state():308] send defer: 9
2021-09-18 16:32:47,581 DEBUG   SenderThread:3618044 [sender.py:send():179] send: final
2021-09-18 16:32:47,581 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: defer
2021-09-18 16:32:47,581 DEBUG   SenderThread:3618044 [sender.py:send():179] send: footer
2021-09-18 16:32:47,581 INFO    HandlerThread:3618044 [handler.py:handle_request_defer():141] handle defer: 9
2021-09-18 16:32:47,582 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: defer
2021-09-18 16:32:47,582 INFO    SenderThread:3618044 [sender.py:send_request_defer():304] handle sender defer: 9
2021-09-18 16:32:47,682 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-18 16:32:47,682 DEBUG   SenderThread:3618044 [sender.py:send_request():193] send_request: poll_exit
2021-09-18 16:32:47,683 INFO    SenderThread:3618044 [file_pusher.py:join():182] waiting for file pusher
2021-09-18 16:32:47,684 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: get_summary
2021-09-18 16:32:47,685 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: sampled_history
2021-09-18 16:32:47,685 DEBUG   HandlerThread:3618044 [handler.py:handle_request():124] handle_request: shutdown
2021-09-18 16:32:47,685 INFO    HandlerThread:3618044 [handler.py:finish():638] shutting down handler
2021-09-18 16:32:48,581 INFO    WriterThread:3618044 [datastore.py:close():288] close: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210918_163150-g9jtz9zf/run-g9jtz9zf.wandb
2021-09-18 16:32:48,683 INFO    SenderThread:3618044 [sender.py:finish():945] shutting down sender
2021-09-18 16:32:48,683 INFO    SenderThread:3618044 [file_pusher.py:finish():177] shutting down file pusher
2021-09-18 16:32:48,683 INFO    SenderThread:3618044 [file_pusher.py:join():182] waiting for file pusher
2021-09-18 16:32:48,685 INFO    MainThread:3618044 [internal.py:handle_exit():78] Internal process exited
